packages/ebayui-core/src/components/ebay-video/index.marko

```diff
@@ -39,14 +53,16 @@ $ const styleSize = {
         <subscribe to=window on-resize("handleResize")/>
     </if>
     <div class="video-player__container" no-update style=styleSize>
-        <video no-controls poster=thumbnail ...processHtmlAttributes(htmlInput)>
-            <for|src| of=sources>
-                <source ...src>
-            </for>
-            <for|track| of=tracks>
-                <track ...track>
-            </for>
-        </video>
+        <if(nav)>
+            <a
+                class="video-player__navigate"
+                ...processHtmlAttributes(nav)>
+                <video-container/>
+            </a>
+        </if>
```

Comment:
We don't need the macro here

We can do something like this...
```
<${nav ? "a" : null} ...processHtmlAttributes(nav)>
   <video no-controls etc>
</>
```
If nav is not set, then video will be unwrapped. 

Also the anchor tag class should be renamed. Maybe `video-player__link`

Is there also a usecase where `nav` can be a button instead of a link? If so we should add support for `as` attribute which can be set to `a` or `button` (default can be anchor)